src/Framework/CustomBuildEventArgs.cs

```diff
@@ -11,8 +11,9 @@ namespace Microsoft.Build.Framework
     /// Arguments for custom build events.
     /// </summary>
     /// <remarks>
-    /// In MSBuild 17.8 and later, this type is deprecated; instead use <see cref="ExtendedCustomBuildEventArgs"/>.
+    /// In .NET 8 and later and Visual Studio 17.8 and later, this type is deprecated; instead use <see cref="ExtendedCustomBuildEventArgs"/>.
```

Comment:
Actually I wonder if we should use one of the fancier ways to do this like https://github.com/dotnet/msbuild/pull/9307#discussion_r1370859586

Reply:
We should!

Reply:
Updated